drivers/Aqara/aqara-lock/profiles/aqara-lock-battery.yml

```diff
@@ -6,6 +6,8 @@ components:
     version: 1
   - id: battery
     version: 1
+  - id: lockAlarm
+    version: 1
   - id: tamperAlert
     version: 1
```

Comment:
It seems like this capability is no longer being used.

Reply:
Actually `tamperAlert` wasn't visible plugin and automation - this is hidden by device-config due to dev request.

Isn't there any issue when removing `tamperAlert` from profile for the existing users?

Reply:
If it's already not visible then there is even less of a concern.

Reply:
@seojune79 Could you remove `tamperAlert` from file with relavant code?

Reply:
I updated it.